agpypeline/transformer_class.py

```diff
@@ -151,7 +149,20 @@ def get_image_file_epsg(self, source_path: str) -> str:
             doesn't have an EPSG code
         """
         # pylint: disable=no-self-use
-        return tr_get_epsg(source_path)
+        logger = logging.getLogger(__name__)
```

Comment:
Logging here needs to be reconciled with the rest of the library. There are now two different approaches with this addition